Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.cpp

```diff
@@ -214,18 +215,18 @@ void RemoteRenderingBackend::didCreateImageBuffer(Ref<ImageBuffer> imageBuffer)
     send(Messages::RemoteImageBufferProxy::DidCreateBackend(WTFMove(*handle)), imageBufferIdentifier);
 }
 
-void RemoteRenderingBackend::moveToSerializedBuffer(RenderingResourceIdentifier identifier)
+void RemoteRenderingBackend::moveToSerializedBuffer(RemoteSerializedImageBufferReference ref)
 {
     assertIsCurrent(workQueue());
     // Destroy the DisplayListRecorder which plays back to this image buffer.
-    m_remoteDisplayLists.take(identifier);
+    m_remoteDisplayLists.take(ref.identifier());
```

Comment:
So as an example: here it is confusing that you destroy the remote image buffer by the reference of a serialized image buffer that doesn't exist. It is confusing.

Reply:
Would it help to create a new type alias with a descriptive name (that is functionally the same type as `RemoteSerializedImageBufferReference`)?
(I'd do this for `RemoteSerializedImageBufferReference` et al.)